components/avatar/avatar_variants.story.vue

```diff
@@ -77,19 +77,40 @@
         <dt-avatar
           :seed="seed"
           full-name="Person avatar"
-          :image-src="imageSrc"
+          clickable
+        />
+        <dt-avatar
+          :seed="seed"
```

Comment:
seed everything here with a static string. Seems to change color on every refresh which will mess up visual tests.

Reply:
Everything is seeded with `seed` string but only on percy visual tests. https://github.com/dialpad/dialtone-vue/blob/d90d8905ef1dfe625140825c42e707fc1f4724e8/components/avatar/avatar.stories.js#L133